Presentation.Web/Controllers/API/V2/Internal/OrganizationUnits/OrganizationUnitRegistrationInternalV2Controller.cs

```diff
@@ -0,0 +1,127 @@
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public IHttpActionResult RemoveRegistrations(Guid organizationUuid, Guid unitUuid, [FromBody] ChangeOrganizationUnitRegistrationV2RequestDTO requestDto)
+        {
+            var changeParameters = ToChangeParameters(requestDto);
+            return _organizationUnitService.DeleteRegistrations(organizationUuid, unitUuid, changeParameters)
+                .Match(FromOperationError, Ok);
+        }
+
+        [HttpPut]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public IHttpActionResult TransferRegistrations(Guid organizationUuid, Guid unitUuid, [FromBody] TransferOrganizationUnitRegistrationV2RequestDTO requestDto)
+        {
+            var changeParameters = ToChangeParameters(requestDto);
+            return _organizationUnitService.TransferRegistrations(organizationUuid, unitUuid, requestDto.TargetUnitUuid, changeParameters)
+                .Match(FromOperationError, Ok);
+        }
+
+        private static OrganizationRegistrationUnitResponseDTO ToRegistrationDto(OrganizationUnitRegistrationDetails details)
+        {
+            return new OrganizationRegistrationUnitResponseDTO
+            {
+                OrganizationUnitRights = details.OrganizationUnitRights.Select(MapUnitRightToNamedEntityDtoWithUserFullNameDto).ToList(),
+                ItContractRegistrations = details.ItContractRegistrations.Select(MapToNamedEntityDTO).ToList(),
+                Payments = details.PaymentRegistrationDetails.Select(ToPaymentRegistrationDto).ToList(),
+                RelevantSystems = details.RelevantSystems.Select(MapToNamedEntityWithEnabledStatusDTO).ToList(),
+                ResponsibleSystems = details.ResponsibleSystems.Select(MapToNamedEntityWithEnabledStatusDTO).ToList()
+            };
+        }
+
+        private static PaymentRegistrationResponseDTO ToPaymentRegistrationDto(PaymentRegistrationDetails details)
+        {
+            return new PaymentRegistrationResponseDTO
+            {
+                ItContract = details.ItContract.MapIdentityNamePairDTO(),
+                ItContractId = details.ItContract.Id,
+                InternalPayments = details.InternalPayments.Select(MapPaymentToNamedEntityDto).ToList(),
+                ExternalPayments = details.ExternalPayments.Select(MapPaymentToNamedEntityDto).ToList()
+            };
+        }
+
+        private static NamedEntityWithUserFullNameV2DTO MapUnitRightToNamedEntityDtoWithUserFullNameDto(OrganizationUnitRight right)
+        {
+            return new NamedEntityWithUserFullNameV2DTO
+            {
+                Id = right.Id,
+                Name = right.Role.Name,
+                UserFullName = right.User.GetFullName()
+            };
+        }
+
+        private static NamedEntityV2DTO MapPaymentToNamedEntityDto(EconomyStream payment)
```

Comment:
I'm not very familiar with this part of the domain, but should these mappers go somewhere else than inside a specific controller so they're reusable elsewhere? Since they map from contracts etc that are also relevant elsewhere.